sda-commons-starter-mongodb/src/main/resources/org/sdase/commons/spring/boot/mongodb/defaults.properties

```diff
@@ -1 +1 @@
-
+de.flapdoodle.mongodb.embedded.version=4.4.1
```

Comment:
Isn't this used only for tests? Shouldn't it be in the test application.properties file?